example_binance_coin_futures.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# File: example_binance_futures.py
+#
+# Part of ‘UNICORN Binance WebSocket API’
+# Project website: https://github.com/oliver-zehentleitner/unicorn-binance-websocket-api
+# Documentation: https://oliver-zehentleitner.github.io/unicorn-binance-websocket-api
+# PyPI: https://pypi.org/project/unicorn-binance-websocket-api/
+#
+# Author: Oliver Zehentleitner
```

Comment:
Not sure about the contribution guideline regarding author here

Reply:
I can add a "Thanks to M3tz3l for contributing" to the example file. Is this ok for you?

Reply:
sure